### DISCUSSION TITLE
How to fetch data by custom query group by

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3035] [Feature Requests]
As we can fetch distinct data from Mysql query by using group by similarly we should be able to fetch data from directus api.
Either by using custom query parameter of any filter property.

ex:-https://abcd.com/abccar-api/items/cars

here are many records of cars.
make: ford , Model: explorer
make: ford , Model: F-150
make: Honda, Model: civic

here I want to fetch distinct car makes

ex: ford,honda

it should not give ford data again.

re: @nileshpande

**Comment by fitu86** [comment]
some like this? 


https://user-images.githubusercontent.com/16139127/130556628-d9b40171-6a5a-4687-9952-15bebf41ca40.mp4

**Reply by rijkvanzanten** [reply]
No, he's referring to a dedicated `group` query parameter that allows for functionality similar to SQL's `GROUP BY` 🙂 This is actually in the works currently: https://github.com/directus/directus/pull/5880

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
